sdk/edgeorder/Azure.ResourceManager.EdgeOrder/src/Generated/EdgeOrderAddressCollection.cs

```diff
@@ -79,19 +80,19 @@ internal static void ValidateResourceId(ResourceIdentifier id)
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentException"> <paramref name="addressName"/> is an empty string, and was expected to be non-empty. </exception>
         /// <exception cref="ArgumentNullException"> <paramref name="addressName"/> or <paramref name="data"/> is null. </exception>
-        public virtual async Task<ArmOperation<EdgeOrderAddressResource>> CreateOrUpdateAsync(WaitUntil waitUntil, string addressName, EdgeOrderAddressData data, CancellationToken cancellationToken = default)
+        public virtual async Task<ArmOperation> CreateOrUpdateAsync(WaitUntil waitUntil, string addressName, EdgeOrderAddressData data, CancellationToken cancellationToken = default)
```

Comment:
Response type missing here.
TSP: https://github.com/Azure/azure-rest-api-specs/blob/c15bc856fc0ec92b150a0c4f80bb9c022046d971/specification/edgeorder/EdgeOrder.Management/AddressResource.tsp#L38

Reply:
did we try to add a `FinalResult = ` in the LROHeader in the corresponding tsp?

Reply:
Fixed